e2e/pages/adf/content_services/search/components/search-radio.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Util = require('../../../../../util/util');
+import { element, by, browser } from 'protractor';
+
+export class SearchRadioPage {
+
+    filter;
+    showMoreButton = element(by.css('adf-search-radio button[title="Show more"]'));
+    showLessButton = element(by.css('adf-search-radio button[title="Show less"]'));
+
+    constructor(filter) {
+        this.filter = filter;
+    }
+
+    checkFilterTypeRadioButtonIsDisplayed(filterName) {
```

Comment:
Please rename checkFilterTypeRadioButtonIsDisplayed to checkFilterRadioButtonIsDisplayed